SDL3/SDL_GetWindowProperties.md

```diff
@@ -125,7 +125,7 @@ This function is available since SDL 3.0.0.
 ## Related Functions
 
 * [SDL_GetProperty](SDL_GetProperty)
-* [SDL_SetProperty](SDL_SetProperty)
+* [SDL_GetNumberProperty](SDL_GetNumberProperty)
```

Comment:
Since this article mentions the properties are readonly, having `SDL_SetProperty` doesn't make sense. I've also added `SDL_GetNumberProperty` as some properties are `*_NUMBER`.